docs/user-guide/prometheus.md

```diff
@@ -0,0 +1,257 @@
+# Prometheus
+
+## Data Model
+
+Data in GreptimeDB is organized as tables, which can be thought of as groups of columns. There are three types of columns: Time Index, Primiary Key, and non of both. When mapping to Prometheus, Time Index is the timestamp, Primary Key is the tag (or label) and the rest are values. Hence, GreptimeDB can be thought of as a multi-value data model, one table is a group of multiple Prometheus metrics. For example:
+
+![Data Model](../public/PromQL-multi-value-data-model.png)
+
+## Storage
+
+GreptimeDB can be used as long-term storage for Prometheus. Using GreptimeDB as a Prometheus backend is a seamless experience. Since Prometheus has built-in support for setting up basic authentication information during the configuration of remote write and read, all you need to do is add your configured username and password to the config YAML file and you're good to go!
+
+Please follow the settings in [Prometheus configuration][https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file] (`prometheus.yml`):
+
+```yaml
+remote_write:
+- url: http://localhost:4000/v1/prometheus/write?db=prometheus
+- basic_auth:
+    username: greptime_user
```

Comment:
maybe we should comment `basic_auth` blocks, because there is no authentication enabled  by default.

Reply:
Maybe we don't need to do that because here is the Guide chapter. People already knew the default no authentication from Getting Started chapter.